File: app/src/main/java/ms/imagine/foodiemate/data/Egg.java
package ms.imagine.foodiemate.data;
/**
 * Created by eugen on 3/30/2018.
 */

import lombok.AllArgsConstructor;
import lombok.Data;
import java.io.Serializable;


@Data
@AllArgsConstructor
public class Egg implements Serializable{
    private String remoteImgURL;
    private int status;
    private long timestamp;

    public Egg(){}
}

File: app/src/main/java/ms/imagine/foodiemate/data/Egg.kt
package ms.imagine.foodiemate.data

import android.net.Uri

class Egg {
    var timestamp: Long = 100
    var status = 0
    var remoteImgURL: String = "noImg"

    constructor(toString: String, toInt: Int, toLong: Long){
        remoteImgURL = toString
        status = toInt
        timestamp = toLong
    }


}